### PR TITLE
Modified to include dynamic color change & custom font

### PR DESCRIPTION
**Pull request by Scraxer**
Two new features added to the config:
image

1) Custom font size  for inventory overlay.
image

2) Dynamically changing color for inventory overlay.
image
image
image
image

CountPlugin.java includes color changing logic.
If the user disables dynamic text color, then the custom text color is chosen instead.

### CODE REVIEW

**Comment by Scraxer** [comment]
@zenrion Hey Zenrion, the only thing I'm not happy with is the font itself, I was intending on allowing selectable fonts OR using the same infoboxFontType but I couldn't quite get things working properly so I defaulted to Arial lol...

**Comment by Scraxer** [comment]
@zenrion 
image

Ignore the custom font & text size in that screeny, was just my testing. Changed the text size into an enumeration, added custom fonts using a default java library & an enum. Added config sections to clearly show related settings/options.

File: src/main/java/io/robrichardson/inventorycount/InventoryCountConfig.java [hunk 1]
```diff
@@ -44,9 +55,61 @@ default Color customInventoryOverlayTextColor() {
...
+    )
+    default InventoryOverlayTextFonts customFont() {
+        return InventoryOverlayTextFonts.ARIAL_BOLD;
+    }
+
+    @ConfigItem(
+            keyName = "customInventoryOverlayFontToggle",
+            name = "Use Custom Font",
+            description = "Toggle using a custom font or the default plugin font",
+            position = 3,
```
Comment (zenrion) [inline src/main/java/io/robrichardson/inventorycount/InventoryCountConfig.java]: Could we make this the first position for this group?

File: src/main/java/io/robrichardson/inventorycount/InventoryCountConfig.java [hunk 2]
```diff
@@ -44,9 +55,61 @@ default Color customInventoryOverlayTextColor() {
...
+    )
+    default InventoryOverlayTextFontSizes customInventoryOverlayFontSize() {
+        return InventoryOverlayTextFontSizes.SIZE_16;
+    }
+
+    @ConfigItem(
+            keyName = "customInventoryOverlayFont",
+            name = "Custom font",
+            description = "Choose a custom font for the inventory counter overlay",
+            position = 1,
```
Comment (zenrion) [inline src/main/java/io/robrichardson/inventorycount/InventoryCountConfig.java]: Could we make this the third position in this group?

**Review by zenrion — CHANGES_REQUESTED** [review]
Just had one tiny change, but everything else looks good!

File: src/main/java/io/robrichardson/inventorycount/InventoryCountConfig.java (hunk 2 above)
Comment (Scraxer) [reply src/main/java/io/robrichardson/inventorycount/InventoryCountConfig.java]: Done!

File: src/main/java/io/robrichardson/inventorycount/InventoryCountConfig.java (hunk 1 above)
Comment (Scraxer) [reply src/main/java/io/robrichardson/inventorycount/InventoryCountConfig.java]: Done!

**Review by zenrion — APPROVED** [review]

**Comment by Scraxer** [comment]
There's a small typo, capital C and F for Custom Font:
image

**Review by zenrion — APPROVED** [review]

**Pull request merged**